test/unit/cog_api/http/relays_test.exs

```diff
@@ -0,0 +1,95 @@
+defmodule CogApi.HTTP.RelaysTest do
+  use CogApi.HTTPCase
+
+  alias CogApi.HTTP.Client
+
+  doctest CogApi.HTTP.Relays
+
+  describe "relay_index" do
+    it "returns a list of relays" do
+      cassette "relay_index" do
+        endpoint = valid_endpoint
+        name = "Index"
+        {:ok, _} = Client.relay_create(%{name: name, token: "1234"}, endpoint)
```

Comment:
Do tokens have to be unique? If so, it might be worth using a different token for each test so these are easier to regenerate